{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -57,6 +57,13 @@ repos:
   hooks:
       - id: black
 
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.990
+  hooks:
+      - id: mypy
+        args: [--no-strict-optional, --ignore-missing-imports]
```

Comment:
minor: I would recommend adding "--disallow-untyped-defs" to make sure we have type hints for function arguments and return values. This will ensure consistency in type annotations across projects.
@stefan-anmut what do you think?

Reply:
@Alighariani yea I think that's a very good idea

Reply:
I've added this option in